P2388.html

```diff
@@ -302,16 +302,18 @@ <h3>Access control</h3>
 <p> Preconditions and postconditions are checked (in <em>Eval_and_abort</em> mode)
     even if the function is called through a pointer.</p>
 
+
 <h3>Side-effects</h3>
 
-<p> Side effects are supported in the predicates of contract annotations.
-    Implementations are allowed to both remove or duplicate these side effects,
-    so they cannot be relied upon. This is similar to the existing rules around eliding copy-constructor side-effects.
```

Comment:
Dropped the analogy with side effect elision, as this has now been replaced with P0135.